packages/compass-indexes/src/modules/regular-indexes.ts

```diff
@@ -73,30 +86,45 @@ type FailedIndexRemovedAction = {
 };
 
 export type State = {
+  isVisible: boolean;
   indexes: RegularIndex[];
-  isRefreshing: boolean;
+  status: FetchStatus;
   inProgressIndexes: InProgressIndex[];
-  error: string | null;
+  error?: string;
```

Comment:
This was string or null here and string or undefined in search indexes. I just picked one and made them the same.